vendor.conf

```diff
@@ -8,11 +8,11 @@ github.com/containerd/containerd 3013762fc58941e33ba70e8f8d9256911f134124
 github.com/containerd/continuity d8fb8589b0e8e85b8c8bbaa8840226d0dfeb7371
 github.com/containerd/fifo fbfb6a11ec671efbe94ad1c12c2e98773f19e1e6
 github.com/containerd/go-runc 4f6e87ae043f859a38255247b49c9abc262d002f
+github.com/containerd/go-cni 575b6a4762a00f6c10d71eb300f07a9d4b5ac695 https://github.com/abhi/go-cni
 github.com/containerd/typeurl f6943554a7e7e88b3c14aad190bf05932da84788
 github.com/containernetworking/cni v0.6.0
-github.com/containernetworking/plugins v0.6.0
+github.com/containernetworking/plugins v0.7.0
```

Comment:
If DAD disable doesn't work, I prefer updating cni plugins later, because the benefit is not clear, and it may introduce new bugs. Kubernetes hasn't validated 0.7.0 plugins yet.

If DAD disable does work, we can try update plugin version to get rid of the mess on our side.

Reply:
sure. did you try with 0.6.0 ? or earlier ?

Reply:
If you try with 0.6.0 without DAD disabled, pod creation will be super slow.

Reply:
Cool ! I am going to make a separate PR for DAD instead of dirtying this